pid_controller_node:
  ros__parameters:
    software_kill_switch_topic: /softwareKillSwitch
    software_operation_mode_topic: /softwareOperationMode
    control_topic: /thrust/wrench_input
    reference_topic: /dp/reference
    odom_topic: /orca/odom
    pose_topic: /dvl/pose
    twist_topic: /dvl/twist
    Kp: [70.0, 70.0, 70.0, 12.0, 12.0, 12.0]
    Ki: [2.0, 2.0, 2.0, 0.12, 0.12, 0.12]
    Kd: [10.0, 10.0, 10.0, 4.0, 5.0, 4.0]
